CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 # Contributing To Simple Icons
 
-Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below.
+Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. If you decide to contribute anything, please do the following:
+
+1. Fork this repository
+1. Hack on a separate topic branch from the latest `develop`
```

Comment:
Should probably be made clearer for people who are new to git. Possibly "Create a new topic branch from the latest `develop`", as GitHub's interface uses "Create branch: [name] (from [base])" when creating new branches through their GUI.

Reply:
In that case I would say we choose for @davidklebanoff suggestion (replacing "topic" by "new"). The GitHub GUI is not the only GUI, and other GUIs may choose a different wording. Agreed?

Reply:
I'm more worried by people understanding the "Hack on a ..." part. If I have no idea what a branch is, "Hack on a separate branch" won't help me understand it at all. "Create a separate branch" indicates that it's an action in and of itself (i.e. you should create your new branch, then start coding). I am indifferent to the topic/new thing - whichever one you guys feel is better is fine by me.

Reply:
:+1: Seems good to me